File: pdlearning/src/microservices/course-microservice/Api/Application/RequestDtos/AnnouncementRequest/SendPlacementLetterRequest.cs
using System;
using System.Collections.Generic;

namespace Microservice.Course.Application.RequestDtos
{
    public class SendPlacementLetterRequest
    {
        public List<Guid> Ids { get; set; }

        public string Base64Message { get; set; }

        public string UserNameTag { get; set; }

        public string CourseTitleTag { get; set; }

        public string CourseCodeTag { get; set; }

        public string CourseAdminNameTag { get; set; }

        public string CourseAdminEmailTag { get; set; }

        public string ListSessionTag { get; set; }

        public string DetailUrlTag { get; set; }
    }
}

File: pdlearning/src/web-apps/src/libraries/domain-api/src/course/dtos/send-placement-letter-request.ts
export interface ISendPlacementLetterRequest {
  ids: string[];
  base64Message: string;
  userNameTag: string;
  courseTitleTag: string;
  courseCodeTag: string;
  courseAdminNameTag: string;
  courseAdminEmailTag: string;
  listSessionTag: string;
  detailUrlTag: string;
}
